AllReadyApp/Web-App/AllReady/Features/Events/EventByMonthQueryHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using AllReady.Models;
+using AllReady.ViewModels.Event;
+using MediatR;
+
+namespace AllReady.Features.Event
+{
+    public class EventByMonthQueryHandler : IRequestHandler<EventByMonthQuery, IEnumerable<EventViewModel>>
+    {
+        private readonly IAllReadyDataAccess dataAccess;
+
+        public EventByMonthQueryHandler(IAllReadyDataAccess dataAccess)
+        {
```

Comment:
@danielepo, the PR has yet to be merged (PR #1131)? I know @MisterJames was working on that. To answer your question of why we're phasing out IAllReadyDataAccess, have a read through Issue #622. The title needs to be updated, but in there are the reasons for the team moving away from the IAllReadyDataAccess abstraction.